README.md

```diff
@@ -53,7 +53,7 @@ module.exports.main = function (page_text_encoded)
 </pre>
 
 
-When releasing your script, you'll need to add the library to a file named "dependencies.txt", located in the root folder of your svn directory. It will need this line:
+When releasing your script, you'll need to add the library to a file named "dependencies.txt", located in the root folder of your svn or git directory. It will need this line:
 <pre>
 https://github.com/Ezandora/Choice-Override/branches/Release/
```

Comment:
this will do an SVN install. You want `github Ezandora/Choice-Override Release`.

Reply:
Thanks,